Makefile

```diff
@@ -98,9 +98,9 @@ test: test_common
 .PHONY: clean
 clean:
 	$(CARGO) clean $(CARGO_MANIFEST)
-	rm $(seabios_blob)
+	rm $(seabios_blob) -f
```

Comment:
What kind of monster puts the flag after the thing you're removing?

Reply:
fixed.